test/e2e/tests/commands/clean.test.ts

```diff
@@ -20,22 +19,11 @@ suite("task - clean", function () {
 
     // Run clean task
     await vscode.commands.executeCommand("workbench.action.tasks.runTask", {
-      task: "hardhat: clean main",
+      type: "hardhat",
+      task: "clean",
     });
     await waitForUI();
 
-    await sleep(1000);
-
-    await vscode.commands.executeCommand(
-      "workbench.action.acceptSelectedQuickOpenItem"
-    );
-    await sleep(1000);
-
-    await vscode.commands.executeCommand(
-      "workbench.action.acceptSelectedQuickOpenItem"
-    );
-
-    // Wait for task to finish
```

Comment:
nit suggestion: keeping the earlier comment `Wait for task to finish`

Reply:
I found the single line of `await new Promise(... onDidEndTask(resolve))` to be pretty self-descriptive but I can add the comment, sure.